charts/bundler/templates/static_config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}-static-data
+  namespace: {{ .Release.Namespace }}
+data:
+  {{ $currentScope := . }}
```

Comment:
The global scope is always available as `$`, so don't need to set it here

Reply:
This was copied from [here](https://helm.sh/docs/chart_template_guide/accessing_files/#glob-patterns)

Reply:
Hmm, no clue why they have that as part of their snippet

Reply:
Without the `currentScope` line (or `with` block), I get 

```
Error: template: bundler/templates/static_config.yaml:10:13: executing "bundler/templates/static_config.yaml" at <.Fil
es.Get>: can't evaluate field Files in type []uint8
```